pkg/parca/parca.go

```diff
@@ -341,7 +343,10 @@ func Run(ctx context.Context, logger log.Logger, reg *prometheus.Registry, flags
 
 	var debuginfodClient debuginfo.DebuginfodClient = debuginfo.NopDebuginfodClient{}
 	if len(flags.Debuginfod.UpstreamServers) > 0 {
-		httpDebugInfoClient, err := debuginfo.NewHTTPDebuginfodClient(logger, flags.Debuginfod.UpstreamServers, flags.Debuginfod.HTTPRequestTimeout)
+		httpDebugInfoClient, err := debuginfo.NewHTTPDebuginfodClient(logger, flags.Debuginfod.UpstreamServers, &http.Client{
+			Transport: promconfig.NewUserAgentRoundTripper(fmt.Sprintf("parca-debugindod-client/%s", version), http.DefaultTransport),
```

Comment:
```suggestion
			Transport: promconfig.NewUserAgentRoundTripper(fmt.Sprintf("parca-debuginfod-client/%s", version), http.DefaultTransport),
```

I would suggest we add the project's URL so it's easier to find

Reply:
How did I miss that typo? 😅 I have just commented on that line. I need glasses 👓

Reply:
oh my I've been making that mistake so often lately ....